docs/use/dapp-staking/for-stakers/index.md

```diff
@@ -0,0 +1,111 @@
+
+For dApps If you get into a tier today, you will start earning rewards tomorrow (next era) and the rewards will be distributed the day after tomorrow.
+
+:::
+
+For stakers, the APR is the same for all stakers, regardless of which dApp they stake on;
+For dApps, staking rewards depends on the Tier system, dApps go into rewards tier based on the amount of tokens staked by users.
+
+Rewards must be claimed before trying to `stake` or `unstake`.
+*(If using the Astar Portal, the interface will automatically claim the rewards when voting)*
+
+If a project is **unregistered** from dApp Staking, staked tokens on that project will still receive **basic rewards** as long as the tokens remain staked.
+
+:::warning
+
+ There is no rewards during the Unbonding period when unstaking from dApp Staking;
+
+ :::
+
+### Bonus Staking Rewards:
+
+If an user stake on dApp(s) during the **Voting Subperiod** and keep the same staked amount or higher on a dApp through the whole **Build&Earn Subperiod**, they will be eligible for the **Bonus Reward** for this dApp.
+
+:::info
+
+**Example**
+
+- User stake 1500 ASTR on **dApp A** and 1000 ASTR on **dApp B** during the `Voting`subperiod.
+- During the `Build&Earn` subperiod, user move 500 ASTR from **dApp B** to **dApp A**;
+- He now has 2000 ASTR on **dApp A** and 500 ASTR on **dApp B**, so he's still eligible for the `Bonus reward` for dApp A but no longer for **dApp B** because the tokens staked on **dApp** B are less than those staked at the end of the `Voting` subperiod (500 < 1000 ASTR).
+- To be eligible for the `Bonus reward` for **dApp B**, the user must stake an additional 500 ASTR on **dApp B** to reach 1000 ASTR.
+
+:::
+
+Bonus rewards for a period can only be claimed after the period finishes.
+
+
+### Other:
+
+- You need to keep a minimum of 10 ASTR or 5 SDN tokens as transferable after staking.
+- **You need to claim to receive your rewards, we recommend claiming your staking rewards once a week.**
+- When unlocking tokens, there is a unbonding period on Shiden and on Astar. The unbonding period lenghts can be consulted [here](/docs/use/dapp-staking/for-stakers/#parameters). Please note that this is based on a perfect block production of 12s. In case of any delay, your unbonding period can be a little longer.
+- The staking rewards are shown in APY by default. APY option is available for stakers who are compounding their rewards by manually re-staking after claiming. On the other hand, APR option is available for stakers who don’t do compounding. The difference is that APR represents the annual rate simply for earning your stake but APY takes compounding into account.
+
+<Figure src={require('/docs/use/dapp-staking/for-stakers/img/13_header.png').default } width="100%" /> 
+
+In case you have any questions, come and join our [Discord channel](https://discord.com/invite/astarnetwork).
```

Comment:
...please check our FAQ section in the Learn part or join our Discord channel...